presto-main/src/main/java/io/prestosql/execution/resourcegroups/InternalResourceGroup.java

```diff
@@ -669,6 +674,32 @@ private void startInBackground(ManagedQueryExecution query)
         }
     }
 
+    public void updateGroupsAndProcessQueuedQueries()
+    {
+        synchronized (root) {
+            updateResourceUsageAndGetDelta();
+
+            while (internalStartNext()) {
```

Comment:
Minor nit:
```
// start as many queries as possible
while (internalStartNext());
```

Reply:
Agreed. But checkStyle won't let me. :) 
I also copied it that way from RootInternalResourceGroup.